lib/ProductOpener/Packaging.pm

```diff
@@ -360,12 +641,40 @@ sub analyze_and_combine_packaging_data ($product_ref) {
 	$log->debug("analyze_and_combine_packaging_data - start", {existing_packagings => $product_ref->{packagings}})
 		if $log->is_debug();
 
+	# Response structure to keep track of warnings and errors
+	my $response_ref = get_initialized_response();
```

Comment:
I find it strange that it's not passed as an argument !
I imagine this function as a small part of a lot of processing like that, maybe combining and putting data in same `$response_ref`.

Reply:
So in fact, this instance of $response_ref is currently not used, we don't do anything with it. We just need one because some of the called functions in it expect one, as we reuse them in other places.

We could move up $response_ref in an argument, all the way to store_product() (which runs all the processing). But then currently we won't use the result.

Reply:
so I'll move the $response_ref = get_initalized_response() in the 3 callers (Import.pm, product_multiliingual.pl and product_jqm_multilingual.pl)

Reply:
and update_all_products.pl